parsers/go.c

```diff
@@ -136,15 +136,13 @@ static tokenInfo *newToken (void)
 	return token;
 }
 
-static tokenInfo *copyToken (tokenInfo *other)
+static void copyToken (tokenInfo *const dest, const tokenInfo *const other)
 {
-	tokenInfo *const token = xMalloc (1, tokenInfo);
```

Comment:
OK, yeah, you're right, that would be possible. On the other hand I searched copyToken() and none of the other parsers creates a token there so maybe better to make it consistent with the rest.